### PR TITLE
TRUNK-5535:Upgrade org.codehaus Libraries

### PR DESCRIPTION
**Pull request by jwnasambu**
## Description of what I changed
I upgrade org.codehaus Libraries

## Issue I worked on
https://issues.openmrs.org/browse/TRUNK-5535

### CODE REVIEW

**Comment by jwnasambu** [comment]
@dkayiwa and @Ruhanga  kindly review my PR at your convenient time.

File: pom.xml [hunk 1]
```diff
@@ -418,12 +418,12 @@
 			<dependency>
 				<groupId>org.codehaus.jackson</groupId>
 				<artifactId>jackson-core-asl</artifactId>
-				<version>1.9.13</version>
+				<version>1.9.13-atlassian-2</version>
```
Comment (dkayiwa) [inline pom.xml]: Why atlassian-2?

Comment (mozzy11) [reply pom.xml]: julie use `<version>1.9.13</version>` , its the latest full release version. 

`<version>1.9.13-atlassian-2</version>` is like a beta release kinda

Comment (HerbertYiga) [reply pom.xml]: @jwnasambu   have you seen the above comments?

**Pull request closed without merging**